packages/shared/components/inputs/RecipientInput.svelte

```diff
@@ -14,30 +15,30 @@
     let inputElement: HTMLInputElement = undefined
     let modal: Modal = undefined
 
-    let selectedAccount: IAccountState
```

Comment:
What was your issue with `selectedAccount`? :D

Reply:
Maybe for consistency I would name it `account` and `onClick(selectedAccount: IAccountState)`. on l.77

Reply:
we use the `selectedAccount`store, so this name is now taken.
i renamed it to `account`